doc/man1/openssl-req.pod.in

```diff
@@ -250,15 +259,31 @@ This option has been deprecated and has no effect.
 
 =item B<-x509>
 
-This option outputs a self signed certificate instead of a certificate
-request. This is typically used to generate a test certificate or
-a self signed root CA. The extensions added to the certificate
-(if any) are specified in the configuration file. Unless specified
-using the B<-set_serial> option, a large random number will be used for
-the serial number.
+This option outputs a certificate instead of a certificate request.
+This is typically used to generate test certificates.
+
+If an existing request is specified with the B<-in> option, it is converted
+to the a certificate; otherwise a request is created from scratch.
+
+Unless specified using the B<-set_serial> option,
```

Comment:
Perhaps move this text to the set_serial option description?

Reply:
I think it's still good to mention this here,
but I've added "If not given, a large random number will be used."
to the description of -set_serial.